view/light.twig

```diff
@@ -148,7 +148,7 @@
       {% include 'right.inc' ignore missing %}
     {% endif %}
     <footer id="footer"></footer>
-  </div>
+  </main>
```

Comment:
This is the closing `div` tag for `div class="main-container"` and should not be changed.

Instead, the closing `div` 6 lines earlier (line 145, just before `{% endblock %}`) should be changed to `</main>`

Reply:
Damnit, I cannot believe I missed that. Fixed, sorry about it.